src/main/resources/reference.conf

```diff
@@ -38,6 +38,12 @@ videobridge {
       }
     }
   }
+  octo {
+    # A string denoting the 'region' of this JVB.  This region
+    # will be used by Jicofo in the selection of a bridge for
+    # a client by comparing it to the client's region.
+    #region
```

Comment:
Add a value to the commented example?
`#region = "region1"`?

Reply:
Sure